packages/server-side-render/src/server-side-render.js

```diff
@@ -62,6 +71,7 @@ export default function ServerSideRender( props ) {
 	const isMountedRef = useRef( true );
 	const fetchRequestRef = useRef();
 	const [ response, setResponse ] = useState( null );
+	const previousResponse = usePrevious( response );
```

Comment:
```suggestion
	const prevResponse = usePrevious( response );
```

Just a nitpick: Maybe we should match the naming pattern here.